src/liger_kernel/ops/cross_entropy.py

```diff
@@ -197,6 +197,7 @@ def liger_cross_entropy_kernel(
             if reduction == "mean":
                 dloss_ori = dloss_ori / sum_non_ignore_weight
                 dloss_smooth = dloss_smooth / sum_non_ignore_weight
+                # z_loss isn't scaled by weight
```

Comment:
Correct me if I were wrong. I guess the comment is to point out the z loss is not scaled by weight, thus it's not divided it by `sum_non_ignore_weight` but `n_non_ignore`. In this case, I think adding a `TODO` for it will be a better idea.

Reply:
Makes sense to me. Thanks for your fast response.